dialogue/TownNPCs/BenchGirlLeft/BenchGirlLeft3.json

```diff
@@ -25,12 +25,21 @@
       "...",
       "De todas maneras, no puedes..."
     ],
+    "pl": [
+      "S-Słyszałeś?\nMówią, że stara wieża zegarowa jest...",
+      "<s>NAWIEDZONA!</s>",
+      "<w>Eeeek!</w>\nNigdy tam nie pójdę, nie ma mowy!",
```

Comment:
Would it be better to use a more phonetic exclamation (for example "Aaaa!") that is more familiar to Polish speakers?

Reply:
I wasn't sure how to translate it, because there's no phonetic exclamation that sounds like "Eeeek" in Polish, but I guess "Aaaa" will also do.